autoemulate/experimental/emulators/lightgbm.py

```diff
@@ -112,17 +80,26 @@ def _fit(self, x: TensorLike, y: TensorLike):
             verbose=self.verbose,
         )
 
+    @staticmethod
+    def is_multioutput() -> bool:
+        return False
+
+    def _fit(self, x: TensorLike, y: TensorLike):
+        """
+        Fits the emulator to the data.
+        The model expects the input data to be:
+            x (features): 2D array
+            y (target): 1D array
+        """
+        x_np, y_np = self._convert_to_numpy(x, y)
+        self.n_features_in_ = x_np.shape[1]
         self.model_.fit(x_np, y_np)
-        self.is_fitted_ = True
 
     def _predict(self, x: TensorLike) -> OutputLike:
         """Predicts the output of the emulator for a given input."""
-        x = check_array(x)
-        # TODO (#422): move to predict() and consider if required
-        check_is_fitted(self, "is_fitted_")
         y_pred = self.model_.predict(x)
-        # Ensure the output is a 2D tensor array with shape (n_samples, 1)
-        return Tensor(y_pred.reshape(-1, 1))  # type: ignore PGH003
+        _, y = self._convert_to_tensors(x, y_pred)  # type: ignore PGH003
```

Comment:
Looking at this again, it seems that it is `spmatrix` (`from scipy.sparse import spamtrix`) and `list` return possibilities we don't suppport. Perhaps instead of ignore we can add?
```python
assert not isinstance(y_pred, (spmatrix, list))
```